src/languages/swift.js

```diff
@@ -291,8 +295,7 @@ export default function(hljs) {
   // https://docs.swift.org/swift-book/ReferenceManual/Expressions.html#ID552
   // Prevents element names from being highlighted as keywords.
   const TUPLE_ELEMENT_NAME = {
-    begin: lookahead(concat(Swift.identifier, /\s*:/)),
-    end: lookahead(/:/),
+    begin: concat(Swift.identifier, /\s*:/),
```

Comment:
I didn't understand why we needed lookahead at all here.

Reply:
This is probably a leftover from when I used `contains`. The goal here was to check that the upcoming code is indeed an identifier, but leave matching to the contained rules.

Will the `keywords: "_|0"` rule still work with this change?

Reply:
_Yes, keywords match the buffer, they do not consume the input._